msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -418,6 +418,49 @@ public async Task<AuthenticationResult> AcquireTokenAsync(IEnumerable<string> sc
                         behavior, extraQueryParameters, parent, ApiEvent.ApiIds.AcquireTokenWithScopeUserBehaviorAuthority).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Non-interactive request to acquire token via Kerberos / Windows Integrated Authentication.
+        /// The username for which the flow is executed is pulled from the operating system, as the current user principal name
+        /// </summary>
+        /// <param name="scopes">Array of scopes requested for resource</param>
+        /// <returns>Authentication result containing token of the current login user</returns>
+        public async Task<AuthenticationResult> AcquireTokenByIntegratedWindowsAuthAsync(IEnumerable<string> scopes)
+        {
+            //TODO: make it platform specific
+            return await AcquireTokenByIWAAsync(scopes, new IWAInput()).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Non-interactive request to acquire token via Windows Integrated Authentication.
+        /// </summary>
+        /// <param name="scopes">Array of scopes requested for resource</param>
+        /// <param name="username">Username for which to run the flow</param>
```

Comment:
Identifier of the user account for which to acquire a token with Integrated Windows authentication. Generally in UserPrincipalName (UPN) format, e.g. john.doe@contoso.com